source/guides/ember-cli/installing.md

```diff
@@ -0,0 +1,62 @@
+## Got Node (and NPM)?
```

Comment:
Now that I've managed to get middleman running I understand @tomdale's comment. By opening with this section it feels more complex than it should be. Maybe move this down to "Troubleshooting" (under a sub-subheader "Installing NPM")? 

As a todo for later: IMO the ember guides are big enough to have a "Troubleshooting" section where we can gather all stuff like this, grouped by category at the very bottom of the Guide sidebar.